src/main/java/com/google/sps/daos/DatabaseGameDao.java

```diff
@@ -84,12 +84,14 @@ public Game getGame(String id) {
       return newGame;
     }
 
+
 @Override
-    public String getNextQuestionId(String gameId, String currentQuestionId) {
+    public String getQuestionId(String question, String gameId, String currentQuestionId) {
```

Comment:
What is the purpose of the first `question` parameter?

Reply:
Oh it's to define if it is looking for the next question or the previous question, because the functions are almost the same, so that parameter defines which one i'm returning.